pkg/istio/rbac/v2/provider_test.go

```diff
@@ -310,6 +321,12 @@ func getFakeOnboardedDomain() zms.SignedDomain {
 									Action:   "post",
 									Effect:   &allow,
 								},
+								{
+									Role:     domainName + ":role.productpage-writer",
+									Resource: domainName + ":svc.productpage:/api/query?*",
+									Action:   "post",
+									Effect:   &allow,
+								},
```

Comment:
could you add one more test case like `?name=test`, case which substring after `?` is not a regex?